src/main/java/com/artipie/api/ArtipieApi.java

```diff
@@ -58,7 +59,7 @@
  * @since 0.6
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
-@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.ExcessiveMethodLength"})
```

Comment:
@olenagerasimova I'd suggest to add a puzzle about some decomposition of constructor, it's hard to read